MapboxNavigation/NavigationViewController.swift

```diff
@@ -200,6 +207,32 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
     
     var bottomViewController: ContainerViewController?
     
+    /**
+     The position of floating buttons in a navigation view. The default value is `MapOrnamentPosition.topTrailing`.
+     */
+    open var floatingButtonsPosition: MapOrnamentPosition? {
+        get {
+            return mapViewController?.floatingButtonsPosition
+        }
+        set {
+            if let newPosition = newValue {
+                mapViewController?.floatingButtonsPosition = newPosition
+            }
+        }
+    }
+    
+    /**
+     The  floating buttons in an array of UIButton in navigation view. The default floating buttons include the overview, mute and feedback report button. The default type of the floatingButtons is `FloatingButton`, which is declared with `FloatingButton.rounded(image:selectedImage:size:)` to be consistent.
```

Comment:
We should make `FloatingButton.rounded(image:selectedImage:size:)` public and document it.

Reply:
Thanks for the suggestion! I'll work on this part.